src/components/Empty.js

```diff
@@ -28,17 +28,17 @@ const Empty = ({
     <EmptyStateBody>{body}</EmptyStateBody>
     {primaryAction && (
       <>
-        {primaryAction.href
-          ? createLink({
-              pathname: `${primaryAction.href}`,
-              linkText: primaryAction.text,
-              history,
-            })
-          : createLink({
-              pathname: `${primaryAction.click}`,
-              linkText: primaryAction.text,
-              history,
-            })}
+        {primaryAction.href ? (
+          createLink({
+            pathname: `${primaryAction.href}`,
+            linkText: primaryAction.text,
+            history,
+          })
+        ) : (
+          <Button variant="link" onClick={primaryAction.click}>
```

Comment:
why move from create link to <button> if the create link ?

Reply:
@acosferreira The problem was when we press on the link of 'create new image' we failed on that row - onClick={() => history.push(pathname)} because the pathname that we pass is a function 'openCreateWizard', so I change it to a button and it solve that bug

Reply:
the bug is also when you try to create new group or new repository (if the table is empty)

Reply:
It looks like the issue is we aren't passing history to Empty so it is undefined

Reply:
i believe instead we should pass the missing parameter

Reply:
This is because we are passing a function and not static url string , we need to pass the action to onClick handler and not to href, handling that in createLink will complicate that function too much, I do not remember the original design , it was a button or link, but in this case we are not modifying the design from earlier

Reply:
E.G we need to handle that onClick function and not to pass it as plain text to href.